tests/cli/test_rasa_init.py

```diff
@@ -1,7 +1,9 @@
 import os
+from typing import Callable, Any, Tuple
+from _pytest.pytester import RunResult
 
 
-def test_init(run):
+def test_init(run: Callable[[Tuple[Any]], RunResult]) -> None:
```

Comment:
I noticed that some tests have a return type and others don't. I think we should decide for one option and stick to that one. On a PR for Rasa X Rick mentioned that he's not including them: https://github.com/RasaHQ/rasa-x/pull/1376#discussion_r334285027.

Reply:
ok, will also remove them then 👍